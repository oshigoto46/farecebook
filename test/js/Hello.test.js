import React from 'react';
import Hello from '../../frontend/components/hello'
import renderer from 'react-test-renderer';

test('Hello', () => {
    const component = renderer.create(
      <Hello />
    )

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
