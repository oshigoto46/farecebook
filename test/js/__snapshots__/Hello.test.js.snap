// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello 1`] = `
<div>
  <p>
    Hello World!
  </p>
</div>
`;
